packaging/Shell/install-ubuntu-deps.sh

```diff
@@ -24,8 +24,7 @@ apt-get -y install \
 	libgraphviz-dev \
 	libsundials-dev
 
-pip3 install --user -r requirements.txt
-pip3 install --user -r requirements-jupyter.txt
+pip3 install --user .
```

Comment:
Any particular reason for the difference here about only pip3 install . vs .[test,docs] in the packaging/Shell/install-fedora-deps-dev.sh

Reply:
@stv0g Here I would also expect only the installation of deps and not the dpsim package.